server/controllers/appointment.js

```diff
@@ -23,6 +27,93 @@ appointmentsRouter.post("/", async (request, response) => {
     response.json(savedAppointment);
 });
 
+// @desc Add Event to Google Calendar
+// @route GET /addevent
+appointmentsRouter.post("/addevent", async (request, response) => {
+    const body = request.body;
+    console.log(body);
+
+    const { userId, eventTypeName, name, timeStart, timeEnd } = body;
```

Comment:
I think in this endpoint besides this line, it doesn't require variable `body` anymore so perhaps you can just call it once here instead of saving it in a variable

```suggestion
    const { userId, eventTypeName, name, timeStart, timeEnd } = request.body;
```